src/ORM/FlexSlider.php

```diff
@@ -205,6 +205,11 @@ public function contentcontrollerInit()
      */
     public function getCustomScript()
     {
+
+        if ($this->owner->config('clear_requirements')) {
```

Comment:
Sorry, one other thing I just noticed. I believe this implementation will always return true, as it will return the full config for `$this->owner`. Updating to `$this->owner->config()->get('clear_requirements')` would have the expected result.